### ISSUE TITLE
Eca_halonext26ts AMP_BF16 training accuracy got failed

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#1275]
### 🐛 Describe the bug

```
python benchmarks/dynamo/timm_models.py --accuracy --amp --amp-dtype bfloat16 -d xpu -n10 --training --only eca_halonext26ts --backend=inductor

xpu  train eca_halonext26ts                   
E0104 12:46:08.845000 967154 site-packages/torch/_dynamo/utils.py:2353] RMSE (res-fp64): 0.01146, (ref-fp64): 0.00222 and shape=torch.Size([512]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.010000, use_larger_multiplier_for_smaller_tensor: 0
E0104 12:46:08.845000 967154 site-packages/torch/_dynamo/utils.py:2223] Accuracy failed for key name stages.3.0.post_attn.running_var
fail_accuracy
```

### Versions

Envirnoments:
Device: PVC 1100
torch-xpu-ops: https://github.com/intel/torch-xpu-ops/commit/18bcd9aaa3353b8a686bde8910c0b818a38cdd79
python: 3.10
TRITON_COMMIT_ID: e98b6fcb8df5b44eb0d0addb6767c573d37ba024
TORCH_COMMIT_ID: b9fbd65dfd5e703bacbc6c25258d1215108b4faf
TORCHBENCH_COMMIT_ID: 766a5e3a189384659fd35a68c3b17b88c761aaac
TORCHVISION_COMMIT_ID: d23a6e1664d20707c11781299611436e1f0c104f
TORCHAUDIO_COMMIT_ID: b6d4675c7aedc53ba04f3f55786aac1de32be6b4
DRIVER_VERSION: 1.23.10.49.231129.50 (803.61)
KERNEL_VERSION: 5.15.0-73-generic https://github.com/intel/torch-xpu-ops/pull/80 SMP Mon May 15 15:18:26 UTC 2023
BUNDLE_VERSION: 2025.0.1.20241113 (DL-Essential 2025.0.1)
OS_PRETTY_NAME: Ubuntu 22.04.2 LTS
GCC_VERSION: 11

**Comment by mengfei25** [comment]
fbnetv3_b  got same issue
```
xpu  train fbnetv3_b
E0104 13:41:14.249000 998978 site-packages/torch/_dynamo/utils.py:2353] RMSE (res-fp64): 0.30015, (ref-fp64): 0.05598 and shape=torch.Size([360]). res.dtype: torch.float32, multiplier: 3.000000, tol: 0.040000, use_larger_multiplier_for_smaller_tensor: 0
E0104 13:41:14.249000 998978 site-packages/torch/_dynamo/utils.py:2223] Accuracy failed for key name blocks.4.3.bn1.running_var
fail_accuracy
```

**Comment by weishi-deng** [comment]
This issue exists on the inductor backend, disabling the decomposition of bn would help.

**Comment by weishi-deng** [comment]
I think this is a natural numeric issue caused by the bf16 type and we can mark it as a known issue:

1. This issue has existed on all backends long before (xpu, cuda, according to weekly tests in 2024 Aug. and Sept.)
2. This accuracy test still fails on CUDA backends in the local test with Pytorch 2.8.
3. The test only fails on bf16 date type and the ut with one single model shape (128, 512, 32, 32) can cause about 0.00674 relative difference on bf16 dtype.

**Comment by weishi-deng** [comment]
> I think this is a natural numeric issue caused by the bf16 type and we can mark it as a known issue:
> 
> 1. This issue has existed on all backends long before (xpu, cuda, according to weekly tests in 2024 Aug. and Sept.)
> 2. This accuracy test still fails on CUDA backends in the local test with Pytorch 2.8.
> 3. The test only fails on bf16 date type and the ut with one single model shape (128, 512, 32, 32) can cause about 0.00674 relative difference on bf16 dtype.

Same as  fbnetv3_b

**Comment by jianyizh** [comment]
Use #1577 to track

**Comment by mengfei25** [comment]
Passed with 63360e64da814de8ce271f1e4b6e2380a03b585e in https://github.com/intel/torch-xpu-ops/actions/runs/15783544727
